README.md

```diff
@@ -2,6 +2,6 @@
 A Minecraft Mod for Forge 1.18 made by Turty's Productions Modding Community with the theme of Ancient/Mythology!
 
 ## Additions:
-- Ore, Raw and Tin Ingot
+- Ore, Raw, Ingot and Tin Block
```

Comment:
This doesn't make any sense. Should be:
`- Tin Ore, Raw Tin, Tin Ingot and Tin Block`

Reply:
Alternatively:
`- Raw Tin, Tin Ore/Ingot/Block`.
But ideally the former would be better.